### PR TITLE
[Examples] set FPS to 60

### PR DESCRIPTION
**Pull request by CrackedPixel**
Made the examples all consistently 60fps. some didnt have any caps, some were non-60

### CODE REVIEW

File: examples/textures/textures_image_rotate.c [hunk 1]
```diff
@@ -45,6 +45,9 @@ int main(void)
     int currentTexture = 0;
     //---------------------------------------------------------------------------------------
```
Comment (raysan5) [inline examples/textures/textures_image_rotate.c:46]: this line should be removed if addded below

File: examples/textures/textures_logo_raylib.c [hunk 2]
```diff
@@ -29,6 +29,9 @@ int main(void)
     Texture2D texture = LoadTexture("resources/raylib_logo.png");        // Texture loading
     //---------------------------------------------------------------------------------------
```
Comment (raysan5) [inline examples/textures/textures_logo_raylib.c:30]: this line should be removed if addded below

File: examples/textures/textures_raw_data.c [hunk 3]
```diff
@@ -65,6 +65,9 @@ int main(void)
     UnloadImage(checkedIm);         // Unload CPU (RAM) image data (pixels)
     //---------------------------------------------------------------------------------------
```
Comment (raysan5) [inline examples/textures/textures_raw_data.c:66]: this line should be removed if addded below

File: examples/textures/textures_to_image.c [hunk 4]
```diff
@@ -40,6 +40,9 @@ int main(void)
     UnloadImage(image);                                    // Unload retrieved image data from CPU memory (RAM)
     //---------------------------------------------------------------------------------------
```
Comment (raysan5) [inline examples/textures/textures_to_image.c:41]: this line should be removed if addded below

**Comment by raysan5** [comment]
@CrackedPixel Just added some small review, please, could you check them?

**Comment by CrackedPixel** [comment]
@raysan5 updated to remove the extra commented out lines. sorry about that!

**Comment by raysan5** [comment]
@CrackedPixel thanks for the reviews!

**Pull request merged**